main.go

```diff
@@ -239,6 +239,7 @@ func start(cfg config.Configuration, addr string, allowedOrigins []string, kubeC
 		controller, err := controllers.FromConfiguration(cfg, sp, authenticator, cl, strg, redirectTpl)
 		if err != nil {
 			zap.L().Error("failed to initialize controller: %s", zap.Error(err))
```

Comment:
can you please also improve this log message? Maybe it's not an error, just a warning. And I would include name of the service provider here, explaining that probably this sp does not have oauth or we don't implement it, or sth like that. This error message will appear on every oauth service startup, so I think it's quite important to be clear here.